constants.properties

```diff
@@ -1,6 +1,6 @@
-gradlePluginsVersion=4.0.0
+gradlePluginsVersion=4.1.0
```

Comment:
4.0.1. It should match the intended version of Corda (eg; 4.0). It's not semver sadly.